packages/lodestar/src/db/api/validator/validator.ts

```diff
@@ -59,7 +59,7 @@ export class ValidatorDB extends DatabaseService implements IValidatorDB {
   }
 
   private incrementPubKey(pubKey: BLSPubkey): string {
-    return new BN(pubKey).addn(1).toString("hex").replace("0x", "");
+    return (toBigIntLE(pubKey) + 1n).toString(16).replace("0x", "");
```

Comment:
I think this should be bytesToBigint from utils so we don't need bigint-buffer dependency